### DISCUSSION TITLE
sig patch wii u sous tiramisu error wup installer

### DISCUSSION THREAD

**Discussion by Thomassss251** [community/community#88515] [Programming Help]
### Body

Bonjour, j’ai un gros problème, j’ai un soucis de sig patch avec la wiiu pour installer certains jeux, auriez vous une solution je débute dans le domaine du hack et je ne veux pas perdre de temps .... ??

Bonjour, j’ai un gros problème, j’ai un problème de patch sig avec la wiiu pour installer certains jeux, avez-vous une solution je suis nouveau dans le domaine du hack et je ne veux pas perdre de temps.... ??

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by CGdiego** [comment]
Hello, you can get Sigpatches Module here: https://github.com/marco-calautti/SigpatchesModuleWiiU/releases.
